File: apps/s-2-admin/src/api/credential/CredentialWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringFilter } from "../../util/StringFilter";
import { JsonFilter } from "../../util/JsonFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { AppModelWhereUniqueInput } from "../appModel/AppModelWhereUniqueInput";
import { DestinationCalendarListRelationFilter } from "../destinationCalendar/DestinationCalendarListRelationFilter";

export type CredentialWhereInput = {
  id?: IntFilter;
  typeField?: StringFilter;
  key?: JsonFilter;
  user?: UserWhereUniqueInput;
  appField?: AppModelWhereUniqueInput;
  destinationCalendars?: DestinationCalendarListRelationFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/Credential/Dtos/CredentialWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class CredentialWhereInput
{
    public int? Id { get; set; }

    public string? TypeField { get; set; }

    public string? Key { get; set; }

    public int? User { get; set; }

    public string? AppField { get; set; }

    public List<int>? DestinationCalendars { get; set; }
}
